easybuild/easyconfigs/z/Zeo++/Zeo++-0.3-intel-2021a.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'MakeCp'
+
+name = 'Zeo++'
+version = '0.3'
+
+homepage = 'https://www.zeoplusplus.org/'
+description = """
+    Zeo++ is a software package for analysis of crystalline porous materials.
+    Zeo++ can be used to perform geometry-based analysis of structure and
+    topology of the void space inside a material,
+    to assemble or alternate structures as well as to generate structure
+    representations to be used in structure similarity calculations.
+    Zeo++ can be used to either analyze a single structure or perform high-throughput analysis of a large database.
+    """
+
+toolchain = {'name': 'intel', 'version': '2021a'}
```

Comment:
It seems like this code only requires a compiler, so `intel-compilers` or `GCC` should be used as toolchain.

The original request asked to be compatible with `intel/2020a`, but that's way too old (see also https://docs.easybuild.io/policies/toolchains).

I'll need to look into installing the `pyiron` module with a more recent toolchain...

Reply:
I change the toolchain to CGG and try out different versions of it (from 10.3.0 to 13.2.0) and it seems it works for all of them. 
For which version of GCC should I do it? Should I make a PR for all the versions?
# toolchain = {'name': 'GCC', 'version': '10.3.0'}
# toolchain = {'name': 'GCC', 'version': '11.2.0'}
# toolchain = {'name': 'GCC', 'version': '11.3.0'}
# toolchain = {'name': 'GCC', 'version': '12.2.0'}
# toolchain = {'name': 'GCC', 'version': '12.3.0'}
# toolchain = {'name': 'GCC', 'version': '13.2.0'}

Reply:
please use `intel-compilers` version `2023.1.0` as toolchain (so `Zeo++` is compatible with `intel/2023a`)

Reply:
toolchain changed to `intel-compilers 2023.1.0`